supplementary/alica_ros1/alica_tests/src/test/Expr/src/conditions/conditions.cpp

```diff
@@ -315,14 +315,6 @@ bool conditionIsAnyChildStatusSuccess3604374027783683696(const Blackboard* input
     return false;
     /*PROTECTED REGION END*/
 }
-bool conditionDecision2B3684268241099966909(const Blackboard* input, const RunningPlan* rp, const Blackboard* gb)
```

Comment:
When conditions are removed from conditions.h / conditions.cpp, they should also be removed from etc/plans/conditions/ConditionRepository.cnd

Also are there no plans using these conditions? I dont see any changes to the plan files.

Reply:
I think the condition still needs to be removed from ConditionRepository.cnd right?